src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -51,14 +62,14 @@ export const styles = StyleSheet.create({
     alignSelf: 'center',
     bottom: 5
   },
-  countUser: {
-    fontFamily: fonts.inter[600],
+  countUser: (from) => ({
     fontSize: normalizeFontSize(14),
     lineHeight: normalizeFontSize(16.94),
-    color: colors.holytosca,
-    marginLeft: 20,
-    marginBottom: 4
-  },
+    color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+    marginLeft: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(4),
+    fontWeight: 'bold'
+  }),
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `countUser` function now accepts a parameter `from` and changes the color based on whether `from` equals `SIGNED`. This change seems to be for better customization of the UI. However, it's important to ensure that the `from` parameter is always provided where this function is used, otherwise it may lead to unexpected behavior.

```diff
-  countUser: {
-    fontFamily: fonts.inter[600],
-     fontSize: normalizeFontSize(14),
-     lineHeight: normalizeFontSize(16.94),
-    color: colors.holytosca,
-    marginLeft: 20,
-    marginBottom: 4
-  },
+  countUser: (from) => ({
+     fontSize: normalizeFontSize(14),
+     lineHeight: normalizeFontSize(16.94),
+    color: from === SIGNED ? colors.darkBlue : colors.holytosca,
+    marginLeft: dimen.normalizeDimen(20),
+    marginBottom: dimen.normalizeDimen(4),
+    fontWeight: 'bold'
+  }),
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->